app/routes.py

```diff
@@ -1,2 +1,76 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, request
+from app.models.planet import Planet
+from app import db
 
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods = ["GET", "POST"])
+def get_planets():
+    if request.method == "GET":
+        planets = Planet.query.all()
+        planets_response = []
+        for planet in planets:
+            planets_response.append({
+                "id": planet.id,
+                "name": planet.name,
+                "description": planet.description,
+                "sign": planet.sign
+            })
```

Comment:
*See comment about helper function to turn instances into dictionaries. We could refactor this line from 5 lines to one by using a helper function

```suggestion
            planets_response.append({planet.to_dict()})
```